hadoop-hdds/client/dev-support/findbugsExcludeFile.xml

```diff
@@ -32,4 +32,8 @@
     <Class name="org.apache.hadoop.hdds.scm.storage.TestBlockInputStream"></Class>
     <Bug pattern="RR_NOT_CHECKED" />
   </Match>
+  <Match>
+    <Class name="org.apache.hadoop.hdds.scm.storage.TestStreamBlockInput"></Class>
+    <Bug pattern="RR_NOT_CHECKED" />
+  </Match>
```

Comment:
This applies to all the places in the code; validate the returned number of bytes read with the expected number of bytes read, and drop the find bugs exception added. Also commented below for the test itself

Reply:
done